<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="MainFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>450</width>
      <height>550</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="tabPosition">
        <enum>QTabWidget::North</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="elideMode">
        <enum>Qt::ElideNone</enum>
       </property>
       <property name="usesScrollButtons">
        <bool>true</bool>
       </property>
       <property name="documentMode">
        <bool>true</bool>
       </property>
       <property name="tabsClosable">
        <bool>false</bool>
       </property>
       <property name="tabBarAutoHide">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="medalTab">
        <attribute name="title">
         <string>Filtre Médailles</string>
        </attribute>
        <widget class="QFrame" name="ButtonFrame">
         <property name="geometry">
          <rect>
           <x>2</x>
           <y>0</y>
           <width>424</width>
           <height>229</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QCheckBox" name="goldCB">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Médailles d'or</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="silverCB">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Médailles d'argent</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="bronzeCB">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Médailles de bronze</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="timeTab">
        <attribute name="title">
         <string>Filtre Temporel</string>
        </attribute>
        <widget class="QFrame" name="mainTimeFrame">
         <property name="geometry">
          <rect>
           <x>3</x>
           <y>4</y>
           <width>422</width>
           <height>238</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <property name="spacing">
           <number>30</number>
          </property>
          <item>
           <widget class="QCheckBox" name="activateTime">
            <property name="text">
             <string>Activer le filtre par temporisation</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="timeFilterFrame">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_10">
             <item>
              <widget class="QFrame" name="filtersFrame">
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Plain</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QFrame" name="leftFilterFrame">
                  <property name="frameShape">
                   <enum>QFrame::NoFrame</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Plain</enum>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_4">
                   <item>
                    <widget class="QRadioButton" name="activateUnder">
                     <property name="text">
                      <string>Filtrer par les années</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="minTLabel">
                     <property name="text">
                      <string>avant:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QSpinBox" name="minTime">
                     <property name="enabled">
                      <bool>false</bool>
                     </property>
                     <property name="autoFillBackground">
                      <bool>false</bool>
                     </property>
                     <property name="accelerated">
                      <bool>true</bool>
                     </property>
                     <property name="minimum">
                      <number>1986</number>
                     </property>
                     <property name="maximum">
                      <number>2016</number>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="Line" name="separator">
                  <property name="lineWidth">
                   <number>1</number>
                  </property>
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QFrame" name="RightFilterFrame">
                  <property name="frameShape">
                   <enum>QFrame::NoFrame</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Plain</enum>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_5">
                   <item>
                    <widget class="QRadioButton" name="activateUpper">
                     <property name="text">
                      <string>Filtrer par les années</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QLabel" name="maxTLabel">
                     <property name="text">
                      <string>après:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QSpinBox" name="maxTime">
                     <property name="enabled">
                      <bool>false</bool>
                     </property>
                     <property name="minimum">
                      <number>1986</number>
                     </property>
                     <property name="maximum">
                      <number>2016</number>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="activateBoth">
               <property name="text">
                <string>Filtrer par un intervalle de temps</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QWidget" name="othersFilters">
        <attribute name="title">
         <string>Autre Filtres</string>
        </attribute>
        <widget class="QFrame" name="otherFilterMainFrame">
         <property name="geometry">
          <rect>
           <x>5</x>
           <y>0</y>
           <width>425</width>
           <height>235</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetNoConstraint</enum>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="lineOutputLimitFrame">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="lineWidth">
             <number>-1</number>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QFrame" name="limitFrame">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QLabel" name="limitLabel">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>12</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>Limite de lignes :</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="limitSpin">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="accelerated">
                   <bool>true</bool>
                  </property>
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="maximum">
                   <number>39765</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="enableLimit">
               <property name="contextMenuPolicy">
                <enum>Qt::NoContextMenu</enum>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Limité le nombre de ligne </string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="sortByFrame">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QFrame" name="filterSortByFrame">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <widget class="QFrame" name="sortByWhatFrame">
                  <property name="frameShape">
                   <enum>QFrame::NoFrame</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Plain</enum>
                  </property>
                  <property name="lineWidth">
                   <number>0</number>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_4">
                   <item>
                    <widget class="QLabel" name="sortByLabel">
                     <property name="font">
                      <font>
                       <pointsize>12</pointsize>
                      </font>
                     </property>
                     <property name="text">
                      <string>Trier par :</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QComboBox" name="sortByCB">
                     <property name="enabled">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QFrame" name="sensFrame">
                  <property name="frameShape">
                   <enum>QFrame::NoFrame</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Plain</enum>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_5">
                   <item>
                    <widget class="QRadioButton" name="ascendingRatio">
                     <property name="enabled">
                      <bool>true</bool>
                     </property>
                     <property name="text">
                      <string>Ascendant</string>
                     </property>
                     <property name="checked">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QRadioButton" name="descendingRatio">
                     <property name="enabled">
                      <bool>true</bool>
                     </property>
                     <property name="text">
                      <string>Descendant</string>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QCheckBox" name="enableSortBy">
               <property name="contextMenuPolicy">
                <enum>Qt::NoContextMenu</enum>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Activer le filtre de tri</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="filterByTeamFrame">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="sizeConstraint">
              <enum>QLayout::SetNoConstraint</enum>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QFrame" name="teamFrame">
               <property name="frameShape">
                <enum>QFrame::NoFrame</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Plain</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>6</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QFrame" name="teamFilterFrame">
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="frameShadow">
                   <enum>QFrame::Raised</enum>
                  </property>
                  <property name="lineWidth">
                   <number>0</number>
                  </property>
                  <layout class="QHBoxLayout" name="horizontalLayout_11">
                   <item>
                    <widget class="QLabel" name="label">
                     <property name="text">
                      <string>Filtre Equipe: </string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QComboBox" name="teamFilterCombo">
                     <property name="enabled">
                      <bool>true</bool>
                     </property>
                     <property name="acceptDrops">
                      <bool>false</bool>
                     </property>
                     <property name="duplicatesEnabled">
                      <bool>false</bool>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="teamActivate">
                  <property name="autoFillBackground">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>Activer le filtre par équipe</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="ResultFrame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPlainTextEdit" name="sqlRequestQT">
          <property name="backgroundVisible">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="ResultButtonFrame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QPushButton" name="updButton">
             <property name="text">
              <string>Mettre à jour la requête</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="confirmButton">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Récupérer les données</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
